_data/sidebars/normalizer_0_7_2_sidebar.yml

```
@@ -66,7 +66,7 @@ entries:
       url: /normalizer_0.7.2_debug_functions.html
       output: web
     - title: Functions Repository
-      url: https://wizzie-io.github.io/normalizer-functions/
+      url: https://wizzie-io.github.io/normalizer-functions_latest_index.html
```

```suggestion
      url: /normalizer-functions_latest_index.html
```

I think that this link can be relative

---

done!, thanks! 😄